.travis.yml

```diff
@@ -6,4 +6,4 @@ install:
 # suppress flag is temporary, will be removed when tests are written
 script:
  - pytest --suppress-no-test-exit-code
```

Comment:
I'm worried about this being in your configs and causing tests to pass that shouldn't be once you start writing them. Go ahead and remove the `--suppress-no-test-exit-code` flag, and instead write a dummy test in your `/tests` directory. Call it `test_dummy.py`, and add a single function `test_dummy` that has a single line `assert 1 == 1`. Then run `pytest` in your repo root to verify this test is running and passing. You'll add more very soon...

Reply:
Added `test_dummy.py` with `test_dummy` function. Ran `pytest` command and "test" successfully ran.